macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -827,10 +827,16 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
                                                              keyValueStore: keyValueStore,
                                                              sessionRestorePromptCoordinator: sessionRestorePromptCoordinator,
                                                              pixelFiring: PixelKit.shared)
-
+#if APPSTORE
+        if AppVersion.runType != .uiTests {
+            updateController = AppStoreUpdateController()
+        }
+#endif
 #if SPARKLE
```

Comment:
Since there's no circumstance in which we'd want both snippets to run, could we use `#elseif SPARKLE` here?